KIPs/kip-11.md

```diff
@@ -0,0 +1,126 @@
+| Community size and engagement level                                                                                                                                | Large, active communities can provide bigger network effects and raise awareness of KyberDMM’s key benefits to more users. Community members from the applying project are highly encouraged to voice their support for the proposal in the forum.                                                                                                                                                                                                                                                                                                                    |
+| Additional value to/Synergy with Kyber Network (marketing, community, product, network effects etc.)                                                               | Partnerships or collaborations should be mutually beneficial and add value to both parties. E.g. integrations that showcase DeFi composability, joint educational activities.                                                                                                                                                                                                                                                                                                                                                                                         |
+| Project tokens need to have a minimum of approximately $200,000 worth of tokens locked in an KyberDMM amplified pool. The pool can be Token-ETH or Token-stablecoin.   | Sufficient initial liquidity for the token pair is required to achieve good rates for traders and generate meaningful volume. Minimum value locked also depends on the AMP (amplification factor) of the pool. Less tokens are required with higher AMP; especially useful for more stable pairs. AMP=1.15 to 1.3 recommended for most non-stable pairs.                                                                                                                                                                                                                                                                                |
+| Ability to at least match the KNC incentives provided by Kyber Network. Minimum of $200,000 worth of incentives in total from both parties. | Both Kyber and the project should commit a meaningful amount of incentives in order to ensure an attractive APY and amplify the impact of the Rainmaker program. The project should indicate a ballpark range of token incentives they are willing to contribute. KNC reward pool will depend on whether it is a new or established token: Kyber can cover from 30%, 50%, or up to 1:1 of the project’s contribution, depending on how established the token is. On top of KNC rewards, Kyber will provide additional market budget and support to promote the campaign. |
```

Comment:
whats "market budget"?

Reply:
Funding from DAO community or from new mining KNC proposal (if it is approved by DAO community). Any suggestions for changing content?